test/alternator/test_return_consumedcapacity.py

```diff
@@ -0,0 +1,83 @@
+# Copyright 2024-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+# Tests for the ReturnConsumedCapacity header
+
+import pytest
+from botocore.exceptions import ClientError
+from decimal import Decimal
+from test.alternator.util import random_string, random_bytes
+
+# A basic test that gets an item from a table with and without consistency
+KB = 1024
+
+def test_basic_get(test_table):
+    p = random_string()
+    c = random_string()
+    val = random_string()
+    val2 = random_string()
+    test_table.put_item(Item={'p': p, 'c': c, 'attribute': val, 'another': val2})
+    response = test_table.get_item(Key={'p': p, 'c': c}, ConsistentRead=True, ReturnConsumedCapacity='TOTAL')
+    item = response['Item']
+    assert item['p'] == p
+    assert item['c'] == c
+    assert item['attribute'] == val
+    assert item['another'] == val2
+    consumed_capacity = response.get('ConsumedCapacity')
+    print(consumed_capacity)
+    assert consumed_capacity
+    capacity_unit = consumed_capacity["CapacityUnits"]
+    assert capacity_unit == 1
```

Comment:
nitpick: no need to name a variable. Can do the above two lines in just one:
```
assert 1 == consumed_capacity["CapacityUnits"]
```
It makes for shorter and clearer code, and also clearer error messages if the test dails..